CONTRIBUTING.rst

```diff
@@ -60,6 +60,33 @@ Once you have done this, you can launch the master branch of Jupyter notebook
 from any directory in your system with::
 
     jupyter notebook
+  
+Verification 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    
+While running the notebook, select one of your notebook files (the file will have the extension .ipynb). 
+In the top tab you will click on "Help" and then click on "About". In the pop window you will see information about the version of Jupyter that you are running. You will see "The version of the notebook server is:". 
+If you are working in development mode, you will see that your version of Jupyter notebook will include the word "dev". 
+
+.. image:: https://user-images.githubusercontent.com/327925/38238574-8a1ada9e-372a-11e8-8a9c-2a74af743390.png
```

Comment:
See how images are added for example in https://github.com/jupyter/notebook/blob/master/docs/source/ui_components.rst

You can't make the assumption that the user have an internet connection when viewing the documentation. Some user will for example build the documentation in a air-gapped network at the DoD. 

You need to add the actual file in something like `docs/source/_static/images/` and then refer to this path. 

This will also ensure that if you build the docs PDF the image will show up !